shared_task_code/results_parser.py

```diff
@@ -55,6 +55,15 @@ def process_results(self, topic, bucket, key):
         result_file_name = re.sub(r"\.tar.gz$", "", key.split("/", -1)[-1])
         body = tar.extractfile(result_file_name).read().decode('utf-8').split('\n')
 
+        # there might be times where we want to handle the results ourselves, if so, pass out
+        # the file and let the scanning task handle it themselves
+
+       
+        if self.func_customdata!=None:
+            #TODO: document this
+            self.func_customdata(body, TarData=tar, MsgData=msgdata, TaskTopic=topic, TaskBucket=bucket, TaskKey=key)
+            
+
         # we need these to track the scan (TODO: this will be removed and replaced by a better identifier later)
         start_time, end_time = msgdata["scan_end_time"], msgdata["scan_end_time"]

```

Comment:
This was a pre-existing copy paste error, but start_time here is assigned the end time value

Reply:
this was intentional as the parent task's end time was to be used for tracking the task through the system for x-ray (until a better identifier was used) - start_time was therefore set to be the same.  as we're not supporting x-ray at the moment, I've now changed the code so that these are actual start/end times for the scans